tests/Service/Parser/Fixtures/DummyFilter.php

```diff
@@ -0,0 +1,53 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\ApiFilter\Service\Parser\Fixtures;
+
+use Lmc\ApiFilter\Entity\Value;
+use Lmc\ApiFilter\Filter\FilterInterface;
+
+class DummyFilter implements FilterInterface
```

Comment:
It is not Dummy at all. It is using for tests. How about `TestingFilter`, `RawFilter`, `SimpleFilter` or somethink else?